src/components/views/investors/SignUp.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Field from '../../common/Fields.jsx';
+
+
+const fields = [
+  {
+    name: 'Full Name', type: 'text', placeholder: 'Please enter your Full Name', label: 'Full Name'
+  },
+  {
+    name: 'phone', type: 'number', placeholder: 'Please enter your Phone Number', label: 'Phone Number'
+  },
+  {
+    name: 'email', type: 'email', placeholder: 'Please enter your Email Address', label: 'Email Address'
+  },
+  {
+    name: 'password', type: 'password', placeholder: 'Please enter your password', label: 'Choose Password'
+  }
+];
+
+class Signup extends Component {
+  render() {
+    return (
+        <div className="login-wrapper signup-wrapper" >
+          <form action="">
+            <div className='login-form-header'>
+              <h2 className='signup-sub'>{this.props.subHeader}</h2>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['subHeader' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)